src/roadmap/releases.md

```diff
@@ -136,8 +136,8 @@ verify it can be fetched (and verified):
 ```
 $ helm repo add controller https://charts.deis.com/controller
 "controller" has been added to your repositories
-$ helm fetch --verify controller/controller --version v2.11.0
-Verification: &{0xc420776190 sha256:97116498fcb5bd9da791d26337694bf64fc20815b3e5c0ae3c543def4404f31e controller-v2.11.0.tgz}
+$ helm fetch --verify controller/controller --version v2.12.0
+Verification: &{0xc420776190 sha256:f64d945a6755c7a0e3165d59a91226dc0fdadb0cc7eac2fb0463c86a74ed3649 controller-v2.12.0.tgz}
```

Comment:
technically the sha is:
```
 $ helm fetch --verify controller/controller --version v2.12.0
Verification: &{0xc420780050 sha256:e5835db3f2131893a70c8df0114f26c6fd8d2520a2dfb232a486b298ce9ad31c controller-v2.12.0.tgz}
```
(controller chart here as opposed to workflow elsewhere... easy trip-up)